src/rhsm/connection.py

```diff
@@ -310,9 +310,12 @@ def __init__(self, code, msg=None, headers=None):
         self.msg = msg or ""
         self.headers = headers or {}
 
+    @property
+    def title(self):
+        return httplib.responses.get(self.code, "Unknown")
+
     def __str__(self):
-        error_title = httplib.responses.get(self.code, "Unknown")
-        return "HTTP error (%s - %s): %s" % (self.code, error_title, self.msg)
+        return "HTTP error (%s - %s): %s" % (self.code, self.title, self.msg)
```

Comment:
Why not f-string here, like in the PR for `main`? The baseline for 1.28 is Python 3.6, which does have them, and they are used in other few bits of this branch already.

Reply:
I wasn't sure, I wanted to play it safe. Fixed.